src/main/java/gov/nasa/jpl/mbee/mdk/MDKPlugin.java

```diff
@@ -95,6 +98,9 @@ public void init() {
         loadExtensionJars(); // people can actually just create a new plugin and
 
         configureEnvironmentOptions();
+
+        SpecificationDialogManager.getManager().addConfigurator(Element.class, new SpecificationNodeAspectsConfigurator());
```

Comment:
Will this cause a new tab in the Specification window? If so, it needs to be removed until that feature is completed.